services/web/client/source/class/qxapp/data/model/Workbench.js

```diff
@@ -300,7 +300,7 @@ qx.Class.define("qxapp.data.model.Workbench", {
       const allNodes = this.getNodes(true);
       const nodes = Object.values(allNodes);
       for (const node of nodes) {
-        if (!node.isDynamic()) {
+        if (node.isComputational() && !node.isInKey("file-picker")) {
```

Comment:
why is the file picker a computational service? it's pretty much dynamic isn't it?